### PR TITLE
SI-399-hide-PII-titles-from-google-analytics

### PR DESCRIPTION
**Pull request by jonhindle**
Add override to GA titles to prevent them from recording prisoner details

### CODE REVIEW

File: server/views/partials/layout.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
       window.dataLayer = window.dataLayer || [];
       function gtag(){dataLayer.push(arguments);}
       gtag('js', new Date());
-      gtag('config', '{{ googleAnalyticsKey }}');
+      gtag('config', '{{ googleAnalyticsKey }}', {{ { title: '' } if googleAnalyticsOverrideTitle else {} }});
```
Comment (Stubbs) [inline server/views/partials/layout.html:15]: Will this switch all analytics off for when override is true?

File: server/views/formPages/approvedView.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {% extends "./formTemplate.html" %}
 {% from "summary-list/macro.njk" import govukSummaryList %}
 
+{% set googleAnalyticsOverrideTitle = true %}
```
[the remark this answers is not in the document]
Comment (jonhindle) [reply server/views/formPages/approvedView.html]: Can do 🤷 
Blanking the title for now seems cleanest; if required we can add in override values later when we've got a clearer idea of what we want to do with the analytics.

File: server/views/partials/layout.html (hunk 1 above)
Comment (jonhindle) [reply server/views/partials/layout.html]: No, it should just override the title value sent to GA.

**Review by Stubbs — APPROVED** [review]
LGTM

**Pull request merged**